frontend/public/components/edit-yaml.jsx

```diff
@@ -585,21 +590,60 @@ export const EditYAML = connect(stateToProps)(
       });
     }
 
+    toggleSidebar = () => {
+      this.setState((state) => {
+        return { showSidebar: !state.showSidebar };
+      });
+      window.dispatchEvent(new Event('sidebar_toggle'));
+    };
+
     render() {
       if (!this.props.create && !this.props.obj) {
         return <Loading />;
       }
 
-      const { connectDropTarget, isOver, canDrop } = this.props;
+      const { connectDropTarget, isOver, canDrop, create, yamlSamplesList } = this.props;
       const klass = classNames('co-file-dropzone-container', {
         'co-file-dropzone--drop-over': isOver,
       });
 
-      const { error, success, stale, yaml, height } = this.state;
-      const { create, obj, download = true, header } = this.props;
+      const { error, success, stale, yaml, height, showSidebar } = this.state;
+      const { obj, download = true, header } = this.props;
       const readOnly = this.props.readOnly || this.state.notAllowed;
       const options = { readOnly, scrollBeyondLastLine: false };
       const model = this.getModel(obj);
+      const samples = getResourceSidebarSamples(model, yamlSamplesList);
+      const showSamples = !_.isEmpty(samples) && create;
+      const sidebarLinkLabel = `View Schema ${showSamples ? 'and Samples' : ''}`;
+      const sidebarLink = (
+        <div className="yaml-editor__link">
+          <Button type="button" variant="link" isInline onClick={this.toggleSidebar}>
+            <InfoCircleIcon className="co-icon-space-r co-p-has-sidebar__sidebar-link-icon" />
+            {sidebarLinkLabel}
+          </Button>
+        </div>
+      );
+      const divider = <div className="yaml-editor__links-divider">|</div>;
+      const shortcuts = (
+        <div className="yaml-editor__link">
+          <Popover
+            aria-label="Shortcuts"
+            bodyContent={
+              <ul>
+                <li>Use Ctrl + Space to active auto complete.</li>
+                <li>Use Ctrl + Shift + O for document outlining.</li>
+                <li>Hover over a key to view a description.</li>
+              </ul>
+            }
+            position="bottom"
```

Comment:
I think it's better to not position the popover so it appears above the YAML editor, thus not overlaying the content. 
 Additionally, we should avoid having a widowed word in each bullet by increasing `maxWidth`.  And to avoid the arrow tip pointing directly at the horizontal divider, we should adjust the `distance`.  Ideally we'd also add `hideOnOutsideClick={false}` so the popover stayed open while editing yaml, but unfortunately, that doesn't appear to work (see https://github.com/patternfly/patternfly-react/issues/3252).
```suggestion
            maxWidth="25rem"
            distance="18"
```
![Screen Shot 2019-11-01 at 10 27 53 AM](https://user-images.githubusercontent.com/895728/68032187-90b9d280-fc93-11e9-9ceb-7cb9cce9f5f1.png)